tcf_website/views/feedback.py

```diff
@@ -18,3 +19,17 @@ def post_message(query):
         url, data=json_data, headers={
             "Content-Type": "application/json"})
     return JsonResponse(content)
+
+
+def send_email(query):
+    """Send email to support"""
+    subject = query.GET.get("subject", "")
+    content = query.GET.get("content", "")
+    from_email = os.environ['EMAIL_HOST_USER']
+    recipient_list = [
+        query.GET.get(
+            "recipients",
+            "support@thecourseforum.com")]
+
+    send_mail(subject, content, from_email, recipient_list)
+    return JsonResponse({'content': query.GET.get("content", "")})
```

Comment:
It's somewhat inconsistent that you use variables for `query.GET.get("subject", "")` and `query.GET.get("content", "")` but not for `query.GET.get("recipients", "support@thecourseforum.com")`. Also, you're not using the variable `content` in the last line.

Reply:
I replaced the query with the `content` variable. The `recipient_list` requires an array instead of a string, so I thought this way would be simpler than passing an array as a query parameter.